### ISSUE TITLE
Ability to Customize Colors

### ISSUE THREAD

**Issue by blackshrub** [BookStackApp/BookStack#1380]
Especially for the page (now blue) and chapter (now orange).

**Comment by ssddanbrown** [comment]
As of BookStack v0.27 the entity (Chapter, Book, Page, Shelve) colors are now set with CSS variables, which means it's fairly easy to change the colors of these via the Custom HTML head content setting:

```html
<style>
	:root {
		--color-page: #a242d7;
		--color-page-draft: #cfb732;
		--color-chapter: #db5382;
		--color-book: #534ecb;
		--color-bookshelf: #3eeaf0;
	}
</style>
```

I'm still considering making these easily alterable via the UI.

**Comment by antimatteraccord** [comment]
> As of BookStack v0.27 the entity (Chapter, Book, Page, Shelve) colors are now set with CSS variables, which means it's fairly easy to change the colors of these via the Custom HTML head content setting:
> 
> ```
> <style>
> 	:root {
> 		--color-page: #a242d7;
> 		--color-page-draft: #cfb732;
> 		--color-chapter: #db5382;
> 		--color-book: #534ecb;
> 		--color-bookshelf: #3eeaf0;
> 	}
> </style>
> ```
> 
> I'm still considering making these easily alterable via the UI.

Please add these to the UI. 
Is it possible to also add an entry so I can change the main backgrounds of every page in this manner as well? That would appease a lot of the 'MOAR DARK MODE' people like myself.

**Comment by ssddanbrown** [comment]
Dark mode would require a lot more than just a change of background colour, Would be an alternative style-sheet.

**Comment by james-geiger** [comment]
@ssddanbrown Is this something that you'd welcome a pr to place in the settings ui?

> As of BookStack v0.27 the entity (Chapter, Book, Page, Shelve) colors are now set with CSS variables, which means it's fairly easy to change the colors of these via the Custom HTML head content setting:
> 
> ```
> <style>
> 	:root {
> 		--color-page: #a242d7;
> 		--color-page-draft: #cfb732;
> 		--color-chapter: #db5382;
> 		--color-book: #534ecb;
> 		--color-bookshelf: #3eeaf0;
> 	}
> </style>
> ```
> 
> I'm still considering making these easily alterable via the UI.

**Comment by ssddanbrown** [comment]
@james-geiger Yeah, sure!

**Comment by ssddanbrown** [comment]
@james-geiger has now added this feature in #1723 and it has been merged into master for the next release (v0.28).

Since this feature is in `master` this issue will now be closed.
